ci/scripts/pr_list_database.py

```diff
@@ -174,42 +141,19 @@ def input_args():
             print(f'Error: {args.dbfile} does not exsist')
             sys.exit(-1)
 
-    con = sql_connection(args.dbfile)
-    obj = con.cursor()
+    ci_database = SQLiteDB(args.dbfile)
+    ci_database.connect()
 
     if args.create:
-        sql_table(obj)
-
+        create(ci_database)
     if args.add_pr:
-        rows = sql_fetch(obj)
-        for row in rows:
-            if str(row[0]) == str(args.add_pr[0]):
-                print(f"pr {row[0]} already is in list: nothing added")
-                sys.exit(0)
-
-        entities = (args.add_pr[0], 'Open', 'Ready', 0, 'ci_repo')
-        sql_insert(obj, entities)
-
+        add_pr(ci_database, args.add_pr[0])
     if args.update_pr:
-        if len(args.update_pr) < 2:
-            print(f"update_pr must have at least one vaule to update")
-            sys.exit(0)
-        pr = args.update_pr[0]
-
-        sql_update(obj, pr, args.update_pr[1:])
-
+        update_pr(ci_database, args)
     if args.remove_pr:
-        sql_remove(obj, args.remove_pr[0])
-
+        remove_pr(ci_database, args.remove_pr[0])
     if args.display is not None:
-        rows = sql_fetch(obj)
-        if len(args.display) == 1:
-            for row in rows:
-                if int(args.display[0]) == int(row[0]):
-                    print(' '.join(map(str, row)))
-        else:
-            for row in rows:
-                print(' '.join(map(str, row)))
-
-    con.commit()
-    con.close()
+        for rows in display(ci_database, args):
+            print(rows)
```

Comment:
if you are going to use `wxflow`, also use `logger` instead of `print`.

Reply:
No because this output is used in the bash script.

Reply:
Ok.